src/mmrelay/matrix_utils.py

```
@@ -501,13 +501,15 @@ def _canonical_server(value: str | None) -> str | None:
 
 def _get_msgs_to_keep_config(config_override: dict[str, Any] | None = None) -> int:
     """
-    Determine how many Meshtastic–Matrix message mappings to retain.
-
-    Prefers the new configuration key `database.msg_map.msgs_to_keep`; falls back to the legacy `db.msg_map.msgs_to_keep` and emits a deprecation warning if that legacy key is used. If no valid integer is configured, returns DEFAULT_MSGS_TO_KEEP. When provided, `config_override` is consulted instead of the module-level `config`.
+    Return the configured number of Meshtastic–Matrix message mappings to retain.
```

Ruff (0.14.11)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)